test/framework/functional/output_http.go

```diff
@@ -0,0 +1,99 @@
+package functional
+
+import (
+	"strings"
+
+	log "github.com/ViaQ/logerr/v2/log/static"
+	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+	"github.com/openshift/cluster-logging-operator/internal/runtime"
+	"github.com/openshift/cluster-logging-operator/internal/utils"
+)
+
+const (
+	//VectorImage          = "quay.io/domino/vector:0.26.X-distroless-static"
+	VectorImage          = "docker.io/timberio/vector:0.25.0-alpine"
```

Comment:
We should:
* move this to someplace where it is not buried here in the tests

@alanconway  any reason we should not use our collector image?

Reply:
> any reason we should not use our collector image?

it is probably because we dont have `http` receiver in our image

Reply:
Is enabling http as  a receiver in the image different then enabling it as a sender?  @alanconway do we want to enable http as a receiver in our image?

Reply:
i  am mistaken. our image in quay has both http sink and http source enabled.

Reply:
updated PR to use same image